README.md

```diff
@@ -219,6 +223,8 @@ The project is now configured for use with the MSDK development repo, and subseq
 
 #### Option 2 - Edit setenv
 
```

Comment:
When does setenv.bat get run?

Reply:
`Tools/Eclipse/cdt/eclipse.bat`, which is what the Start Menu shortcut points to:

```batch
@echo off

pushd ..\..\..\
call setenv.bat
popd

start eclipse.exe
```

Reply:
That's what I was thinking, but the instructions to launch eclipse.bat were removed above.  Were those instructions added the UG instead?

Reply:
Ah sorry, yes see https://analog-devices-msdk.github.io/msdk/USERGUIDE/#running-eclipse